codegen/core/src/test/scala/com/lightbend/akkasls/codegen/ModelBuilderSuite.scala

```diff
@@ -83,13 +85,16 @@ class ModelBuilderSuite extends munit.FunSuite {
 
       val googleEmptyProto =
         PackageNaming(
+          "Empty",
```

Comment:
Not important for the tests, but could be better to give an `"empty.proto"` filename?

Reply:
That was what I started with, but then there were many test failures, so I changed to what the tests were expecting. I think there is a special case for the Empty.